Makefile

```diff
@@ -241,7 +242,10 @@ libceed.c += $(blocked.c)
 libceed.c += $(opt.c)
 
 # Memcheck Backend
-ifneq ($(shell which valgrind 2> /dev/null),)
+MEMCHK_STATUS = Disabled
+MEMCHK := $(shell if echo "\#include <valgrind/memcheck.h>" | $(CC) -x c -c - >/dev/null 2>&1; then echo 1; fi;)
```

Comment:
This leaves behind a file `-.o`, but it should be sufficient to `$(CC) -E -`, which does not. Pushed this update.